README.md

```diff
@@ -1,24 +1,185 @@
-# README
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [:card_index: Entity Relationship Diagram](#er-diagram)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-Things you may want to cover:
+<!-- PROJECT DESCRIPTION -->
 
-* Ruby version
+# 📖 Budget App
+## 🛠 Built With <a name="built-with"></a>
 
-* System dependencies
+- Ruby on Rails
+- Bootstrap
+- JQuery
+- Ruby
+- Rubocop
+- Gems
+  - Rspec
+  - Capybara
+  - Selenium-webdriver
+  - Bcrypt
+  - cancancan
+  - Devise
+  - FactoryBot
 
-* Configuration
+### Tech Stack <a name="tech-stack"></a>
 
-* Database creation
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML, CSS</li>
+    <li>JavaScript</li>
+  </ul>
+</details>
 
-* Database initialization
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
 
-* How to run the test suite
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+<!-- Features -->
 
-* Deployment instructions
+### Key Features <a name="key-features"></a>
 
-* ...
+- **[Create and run necessary migration files]**
+- **[Match table and column names to the ERD diagram.]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ER DIAGRAM-->
+
+## :card_index: Entity Relationship Diagram <a name="er-diagram"></a>
+
+The end result should follow the following data model (this is an Entity Relationship Diagram that you are already familiar with):
+
+> ![](https://github.com/microverseinc/curriculum-rails/raw/main/capstone/images/erd_diagram.png)
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+[video link ](https://www.loom.com/share/8b236ec2435c4fd7940cec5fa2c81c7a)
```

Comment:
[REQUIRED] Kindly include. the link to your deployed project as instructed by the previous reviewer. You should have deployed your project to Netflix or Heroku.

Reply:
Dear fobadara,

Heroku or Netflix are not free platform. This options have been tried before now. see heroku pricing list https://www.heroku.com/pricing